frontend/src/components/auth/Credentials.tsx

```diff
@@ -98,9 +98,7 @@ const Credentials = ({
     return <Redirect to={HOME_PAGE} />;
   }
 
-  if (toggle) {
-    // Lock scroll
-    document.body.style.overflow = "hidden"
+  if (toggle) {   
     return (
       <Flex h="100vh">
         <Box w="47%">
```

Comment:
The total width of the items in this Box tag is greater than 100% so it is causing overflow. Changing line 161 to `width="64%` fixed the issue for me.